### DISCUSSION TITLE
Evaluations Not Available in Self-Hosted Version?

### DISCUSSION THREAD

**Discussion by c0pper** [langfuse/langfuse#2130] [Support] [answered]
Description:
I've recently set up the self-hosted version of Langfuse using Docker. I pulled the latest image (langfuse/langfuse:latest) and successfully deployed it. However, I do not see the "Evaluations" option in the sidebar, which I believe should be available based on the documentation.

Steps to Reproduce:

    Deploy the Langfuse container using the latest Docker image:

    sh

    docker run --name langfuse \
    -e DATABASE_URL=postgresql://postgres:password@db/langfuse_db \
    -e NEXTAUTH_URL=http://localhost:3000 \
    -e NEXTAUTH_SECRET=mysecret \
    -e SALT=mysalt \
    --link postgres-langfuse-db-1:db \
    -p 3000:3000 \
    langfuse/langfuse:latest

    Access the Langfuse web UI at http://localhost:3000.
    The "Evaluations" option is missing from the sidebar.

Expected Behavior:
The "Evaluations" option should be visible in the sidebar, allowing access to the evaluation features.

Actual Behavior:
The "Evaluations" option is not present in the sidebar.

Screenshot:
![image](https://github.com/langfuse/langfuse/assets/18085925/56201eb1-79af-4c12-84a1-4c34bdb2c2c2)

Environment:

    Docker image: langfuse/langfuse:latest
    PostgreSQL version: 12
    Running on: Docker on Windows

Additional Information:
Could you please confirm if the "Evaluations" feature is enabled in the self-hosted version of Langfuse? If it is, are there any additional configurations required to enable it?

Thank you for your assistance!

**Comment by marcklingen** [comment, marked as answer]
This evaluations tab (online model-based evaluation) is currently in public beta on Langfuse cloud ([docs](https://langfuse.com/docs/scores/model-based-evals)). It uses some preview infrastructure of Langfuse v3, see more on this here: https://github.com/orgs/langfuse/discussions/1902

Langfuse model-based evaluations will be part of the commercial version of Langfuse when self-hosting. Learn more about this here: https://langfuse.com/docs/deployment/feature-overview
